jetty-http/src/main/java/org/eclipse/jetty/http/MetaData.java

```diff
@@ -100,6 +101,28 @@ public Iterator<HttpField> iterator()
         return _fields.iterator();
     }
 
+    @Override
+    public int hashCode()
+    {
+        return Objects.hash(_httpVersion, _contentLength, _fields, _trailerSupplier);
+    }
+
+    @Override
+    public boolean equals(Object obj)
+    {
+        if (!(obj instanceof MetaData))
+            return false;
+
+        MetaData other = (MetaData)obj;
+        if (!Objects.equals(_httpVersion, other._httpVersion))
+            return false;
+        if (!Objects.equals(_contentLength, other._contentLength))
+            return false;
+        if (!Objects.equals(_fields, other._fields))
+            return false;
+        return _trailerSupplier == null && other._trailerSupplier == null;
+    }
```

Comment:
A bit strange these changes are in this PR.... but good additions.

Reply:
This was for a test case I wrote comparing metadata but it didn't work out and I got rid of the test case and thought I might as well keep this.